pydash/pydash_app/dashboard/aggregator.py

```diff
@@ -0,0 +1,58 @@
+from collections import defaultdict
+
+import persistent
+
+
+class Aggregator(persistent.Persistent):
+    """
+    Maintains aggregate data for either a dashboard or a single endpoint.
+    This data is updated every time a new endpoint call is added.
+    """
+
+    def __init__(self, endpoint_calls):
```

Comment:
I think it would make sense to give endpoint_calls a default value (as an empty list).

Also, input checking (making sure `endpoint_calls` is actually a list) might be smart here :-)